pkg/spanner/client.go

```diff
@@ -143,10 +144,15 @@ func (c *Client) TruncateAllTables(ctx context.Context) error {
 		}
 	}
 
-	_, err = c.spannerClient.ReadWriteTransaction(ctx, func(ctx context.Context, txn *spanner.ReadWriteTransaction) error {
-		return txn.BufferWrite(m)
-	})
-	if err != nil {
+	eg := errgroup.Group{}
+	for _, stmt := range stms {
+		stmt := stmt
+		eg.Go(func() error {
```

Comment:
With `errgroup.Group`, we can recognize only the first error returned by eg.Go's argument (function). So, it's better to handle all errors by using multierror: https://github.com/hashicorp/go-multierror

Reply:
fixed and used `sync.WaitGroup` instead of `errgroup.Group`.